source/administration/create-new-workspace.rst

```diff
@@ -0,0 +1,18 @@
+Create a New Workspace
+----------------------
+
+To create a new workspace, provide your work email address and create a password. Next, name your workspace.
+
+- We suggest workspace names based on the email address you enter. 
+- If you're using a corporate email address, we use the email domain to suggest a workspace name. 
```

Comment:
We actually suggest a workspace name based off the string entered in Company Name. If it's not available then we try to suggest a name based on the domain of the email address.